apps/emqx_plugins/src/emqx_plugins.erl

```diff
@@ -1376,23 +1409,23 @@ prune_backup_files(Path) ->
         Deletes
     ).
 
-read_file_fun(Path, ErrMsg, #{read_mode := ?RAW_BIN}) ->
+read_file_fun(Path, Msg, #{read_mode := ?RAW_BIN}) ->
```

Comment:
it's better to keep `msg => "the_msg_string"` close to `?SLOG` macro.
I would recommend to simply `throw(Reason)` from this function, and keep Msg outside.

Reply:
will change it in refactor PR (in 5.8.0)